src/cmd.ts

```diff
@@ -123,6 +123,17 @@ function createCommandWithSharedOptions(name: string, description: string) {
 				.choices(Object.values(MatchMode))
 				.makeOptionMandatory(),
 		)
+		.option(
+			"--no-skip-recheck",
+			"Recheck every torrent before resuming. Leave this off to only recheck those that are necessary.",
+			fallback(fileConfig.skipRecheck, true),
+		)
```

Comment:
I'm not too confident with these either, but it worked from my testing iirc. The goal is to default to `skipRecheck: true`. There is no way to prevent resuming, at 0 complete matches will still be resumed.

This option just allows the user to also recheck these first. In the unlikely case where this torrent was corrupted, it won't have 0 remaining and will not resume. In the future, I would like to delete these torrents but I feel that would make this PR too complicated for how rare it is.

The idea is that a user can prevent failures later on in client once the corrupted piece is finally requested by using `skipRecheck: false`. At the cost of incuring tons of rechecking.